psutil/__init__.py

```diff
@@ -744,6 +744,15 @@ def cpu_affinity(self, cpus=None):
             else:
                 self._proc.cpu_affinity_set(list(set(cpus)))
 
+    # Linux and OSX only
+    if hasattr(_psplatform.Process, "environ"):
+        def environ(self):
+            """The process environment variables as a dict.
+            Note: this usually reflects the state when the process was started.
+            Later changes done by the process might not show up here.
+            """
```

Comment:
I don't think it's that easy.  The memory for the environment variables is owned by the process which can do whatever it wants with it as far as I know.  I looked at a couple of setenv implementations and Linux's libcs seem to leave the original memory in place.  It seems that BSD might reuse the memory if you're overwriting a value with a smaller one.  Finally, you can also change the memory returned by getenv.  I only added this note because one of your comment here https://code.google.com/p/psutil/issues/detail?id=52#c18